src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static int bound = 100;
+    private ColorSupplier colorSupplier;

```

Comment:
The `colorSupplier` field is declared but not initialized. Consider initializing it in the constructor or directly in the field declaration to avoid potential `NullPointerException` when calling `getRandomColor()`.